shakenfist/daemons/net.py

```diff
@@ -66,8 +66,7 @@ def _maintain_networks(self):
                     if not inst:
                         stray = True
                     else:
-                        dbstate = inst.state
-                        if dbstate.get('state') in ['deleted', 'error', 'unknown']:
+                        if inst.state.state in ['deleted', 'error', 'unknown']:
```

Comment:
You're right, inst.state.state is a bit weird.

Reply:
Weirdness reduced